ci/conda/build.sh

```diff
@@ -0,0 +1,18 @@
+# Configure step
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DCMAKE_PREFIX_PATH=$PREFIX \
+ -DCMAKE_SYSTEM_PREFIX_PATH=$PREFIX \
+ -DSMESH_TESTING=ON \
+ .
+
+# Build step
+make -j 4
+
+# test
+make test
+
+# Install step
```

Comment:
@tpaviot , could this be the issue of the failed osx built?
i guess running `make install` is required to link the libraries
let me check